test.ts

```diff
@@ -64,14 +64,7 @@ Deno.test("dir", async () => {
 });
 
 Deno.test("unzip", async () => {
-  const dir = await Deno.makeTempDir();
   await exampleZip("example.zip");
-  const z = await readZip("example.zip");
-  await z.unzip(dir);
-
-  const content = await Deno.readFile(join(dir, "Hello.txt"));
-  assertEquals("Hello World\n", decode(content));
-
-  const smile = await Deno.readFile(join(dir, "images", "smile.gif"));
-  assertEquals("", decode(smile));
+  const z = new JSZip();
+  await z.unzip("example.zip");
```

Comment:
this test no longer does anything?

Reply:
just meant to make it clean and focus on its function

Reply:
The problem is this test no longer tests anything.

Reply:
Previously it:

- created a temp dir
- unzipped example.zip into that dir
- asserted the presence of two files (one nested)